docs/lab/aws/ubuntu-ooto-driver.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: ooto-ci-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: ooto-ci-a 
+
+      kernelMappings:
+      #Matching kernel with this literal will trigger build unless same image and version exists.
+        - literal: 5.4.0-1089-azure 
+          containerImage: quay.io/ebelarte/kmmo-lab:1.0-ooto-aws 
+          build:
+            pull:
+              insecure: false
+            push:
+              insecure: false
+            ## We can set up kaniko version to build module image
+            #kanikoParams:
+            #  image: "gcr.io/kaniko-project/executor"
+            #  tag: "debug"
+            dockerfile: |
+              ARG KERNEL_VERSION='5.4.0-1089-azure'
+              FROM ubuntu:18.04 as builder
+              RUN apt-get update && apt-get install -y bc bison flex libelf-dev gnupg wget git make bc gcc
+              RUN apt-get -y install linux-headers-5.4.0-1089-azure
+
+              WORKDIR /usr/src
+              RUN git clone https://github.com/enriquebelarte/ooto-kmod.git
+
+
+              WORKDIR /usr/src/ooto-kmod
+              RUN make 
+
+    
+              FROM ubuntu:18.04 
+              RUN apt-get update && apt-get install -y kmod
+              ARG KERNEL_VERSION
+              RUN mkdir -p /opt/lib/modules/${KERNEL_VERSION}
+              COPY --from=builder /usr/src/ooto-kmod/ooto_ci_a.ko /opt/lib/modules/${KERNEL_VERSION}/
+              COPY --from=builder /usr/src/ooto-kmod/ooto_ci_b.ko /opt/lib/modules/${KERNEL_VERSION}/
+              RUN depmod -b /opt
+  imageRepoSecret: 
+      name: quay-eb   
```

Comment:
Maybe we can come with a better name? maybe something like `quay-image-repo-secret`